www/showuser

```diff
@@ -523,6 +524,9 @@ function pingMultiUserSession(sid, gameid, title, author, started)
                     . "<i>$desc</i></span></div>";
             }
         }
+        
+        echo "<p><span class=details><a href=\"search?authorid=$uid\">"
+            . "See all of this member's games</a></span>";

```

Comment:
I think we only want to show this link if there are more games to see.

I also think it might be nice to show a count and the author's name in this link, e.g. "See all 7 games by Author Name"

Reply:
Okay, I was originally going to fix all of the "See all..." links on the page as a separate issue, but if you want me to push it into this pull request, I can do that.

I like the count idea as well.